src/libraries/Common/src/System/Security/Cryptography/SP800108HmacCounterKdfImplementationCng.cs

```diff
@@ -202,6 +202,12 @@ private static int GetHashBlockSize(string hashAlgorithmName)
                 case HashAlgorithmNames.SHA384:
                 case HashAlgorithmNames.SHA512:
                     return 1024 / 8;
+                case HashAlgorithmNames.SHA3_256:
+                    return 1088 / 8;
+                case HashAlgorithmNames.SHA3_384:
+                    return 832 / 8;
+                case HashAlgorithmNames.SHA3_512:
```

Comment:
I left a comment elsewhere around testing - do we have tests that cover multiple blocks for each of these?

Reply:
Yes. We specifically test around the block boundaries here:

https://github.com/dotnet/runtime/blob/b3efe82cca726ca96d082b48c9de1a22cb64b2ad/src/libraries/Common/tests/System/Security/Cryptography/SP800108HmacCounterKdfTests.Functional.cs#L98-L109